examples/misc/histogram.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
```

Comment:
I've seen a lot of flickering in the histogram when moving the map

Reply:
Yes, because it is updated on every layer update. Victor already mentioned that, so I am going to se how to debounce it.

Reply:
I don't think we should debounce, that would only mitigate the problem and it would look like CVL was slow and debouncing its data. I think that the animation should be removed or be set in a way that was done from the previous state to the new one, not from the zero-state to the new one.